### PR TITLE
Add json to tracker stats

### PR DESCRIPTION
**Pull request by jakefb**

### CODE REVIEW

**Comment by yciabaud** [comment]
Nice!

File: server.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ function Server (opts) {
         return count
       }
 
-      if (req.method === 'GET' && req.url === '/stats') {
+      if (req.method === 'GET' && req.url === '/stats' || 'stats.json') {
```
Comment (yciabaud) [inline server.js]: I believe something is missing here.

File: server.js [hunk 2]
```diff
@@ -185,13 +185,37 @@ function Server (opts) {
...
+          res.end('<h1>' + torrents + ' torrents (' + activeTorrents + ' active)</h1>\n' +
+            '<h2>Connected Peers: ' + peersAll + '</h2>\n' +
+            '<h3>Peers Seeding Only: ' + peersSeederOnly + '</h3>\n' +
+            '<h3>Peers Leeching Only: ' + peersLeecherOnly + '</h3>\n' +
+            '<h3>Peers Seeding & Leeching: ' + peersSeederAndLeecher + '</h3>\n' +
+            '<h3>IPv4 Peers: ' + peersIPv4 + '</h3>\n' +
+            '<h3>IPv6 Peers: ' + peersIPv6 + '</h3>\n')
+        }
+
+        if (req.url === '/stats.json') {
```
Comment (yciabaud) [inline server.js]: Maybe we can handle content-type json in request on /stats endpoint?

**Comment by yciabaud** [comment]
I am interested in merging your PR, I am making the changes I proposed and writing a unit test.

Thank you.

**Comment by yciabaud** [comment]
Here it is, thank you again  @jakefb . @feross is it okay for you if I handle non core PRs on my own?

**Comment by feross** [comment]
Yeah, if there's no response from me for a while, it's fine to merge things. 👍 That's the power of being a collaborator.

If there's something that's likely to be controversial, or something that you're not confident about (maybe not familiar with that part of the codebase, or it's extra tricky), then I would wait for another collaborator to say it looks good first. But you can use your best judgement.

Btw, I left some comments on your followup commit https://github.com/yciabaud/bittorrent-tracker/commit/d7a651f3605056e6e1d09f4bedf0b5f022d2249d

**Pull request merged**